jax/random.py

```diff
@@ -640,6 +672,17 @@ def normal(key: jnp.ndarray,
 
 @partial(jit, static_argnums=(1, 2))
 def _normal(key, shape, dtype) -> jnp.ndarray:
+  if dtypes.issubdtype(dtype, np.complexfloating):
+    key_re, key_im = split(key)
+    dtype = dtypes.dtype_real(dtype)
+    _re = _normal_real(key_re, shape, dtype)
+    _im = _normal_real(key_im, shape, dtype)
+    return 1/jnp.sqrt(jnp.array(2, dtype=dtype))*(_re + 1j*_im)
```

Comment:
Nit: The rest of this code base uses spaces around binary operators. Do the same for consistency. Same everywhere.